README.rst

```diff
@@ -79,7 +79,7 @@ Here's how easy it is to write "Hello World" in CherryPy:
     cherrypy.quickstart(HelloWorld())
 
 And it continues to work that intuitively when systems grow, allowing
-for the Python object model to be dynamically presented as a web site
+for the Python object model to be dynamically presented as a website
```

Comment:
I was unsure, but it does appear "web site" and "website" are acceptable, but [website is preferred](https://lisagoldresearch.wordpress.com/2010/04/17/ap-stylebook-surrenders-the-battle-over-web-site-vs-website/).